docs/concepts/microfrontends.md

```diff
@@ -0,0 +1,66 @@
+---
+parent: Concepts
+---
+
+## ESM micro frontends in Modular
+
+Micro frontends are a pattern where discrete UIs (frontends) are composed from
+independent fragments that can be built and deployed separately by different
+teams and loaded on-demand at runtime. Modular gives developers the opportunity
+of implementing microfrontends through [Views](../building-apps/esm-views.md),
+which are applications built as ES Modules, that can be served standalone or
+dynamically imported by an host application. This is particularly useful when a
+View exports a default React component: an host application can import it at
+runtime and render it in its own React tree, without the need of using Iframes
+and with automatical dependency de-duplication, thanks to the use of an external
```

Comment:
*automatic

Suggestion: "thanks to how we offload dependencies to an external ESM CDN"